### PR TITLE
docs(secret-manager): add tutorial to generate password int-add-secret-manager-generate-password

### PR DESCRIPTION
**Pull request by luxifer**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Add tutorial to generate a password with the Scaleway Secret Manager

### CODE REVIEW

File: tutorials/secret-manager-generate-password/index.mdx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    This will generate a password of length 12 and store it as a new version of the created secret.
+
+3. Get the generated password with the following command:
+    ```
+    scw secret version access secret-id=<the created secret ID> revision=latest
+    ```
+
+    This will print the base64 encoded password.
+
+## With the Scaleway go SDK
```
Comment (nerda-codes) [inline tutorials/secret-manager-generate-password/index.mdx:48]:
```suggestion
##  Generate a password with the Scaleway GO SDK
```

File: tutorials/secret-manager-generate-password/index.mdx [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+<Macro id="iam-requirements" />
+
+With the following code, you can create a password and generate a password as a new version of that secret.
```
Comment (nerda-codes) [inline tutorials/secret-manager-generate-password/index.mdx:52]:
```suggestion
The following code allows you to create a Scaleway account, a secret and to generate a password as a new version of that secret.
```

Comment (luxifer) [reply tutorials/secret-manager-generate-password/index.mdx]: This is an example code to create a secret with the SDK and then generate a password as a new version

File: tutorials/secret-manager-generate-password/index.mdx [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    scw secret secret create project-id=<your project ID> name=test-password
+    ```
+    Copy the secret ID that is shown in the output of the command.
+
+2. Run the following command to generate the password for your newly-created secret. Make sure you replace `secret-id` with the ID of your secret.
+    ```
+    scw secret version generate-password secret-id=<the created secret ID> length=12
+    ```
+
+    This generates a password of 12 characters. Your password is then stored in Secret Manager as a new [version ](/identity-and-access-management/secret-manager/concepts/#version) of your secret.
```
Comment (RoRoJ) [inline tutorials/secret-manager-generate-password/index.mdx:40]:
```suggestion
    This generates a password of 12 characters. Your password is then stored in Secret Manager as a new [version](/identity-and-access-management/secret-manager/concepts/#version) of your secret.
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**